financial_assistant/README.md

```diff
@@ -0,0 +1,150 @@
+- **Generate PDF Report**: Generate a PDF report based on the saved answered queries
+    or on the whole chat history.
+- **Print Chat History**: Print the whole chat history.
+
+# Before you begin
+
+You have to set up your environment before you can run or customize the starter kit.
+
+## Clone this repository
+
+Clone the starter kit repo.
+
+```bash
+git clone https://github.com/sambanova/ai-starter-kit.git
+```
+
+## Set up the models, environment variables and config file
+
+### Set up the generative model
+
+The next step is to set up your environment variables to use one of the inference models available from SambaNova. You can obtain a free API key through SambaNova Cloud. Alternatively, if you are a current SambaNova customer, you can deploy your models using SambaStudio.
+
+- **SambaNova Cloud (Option 1)**: Follow the instructions [here](../README.md#use-sambanova-cloud-option-1) to set up your environment variables.
+    Then, in the [config file](./config.yaml), set the llm `api` variable to `"sncloud"` and set the `select_expert` config depending on the model you want to use.
+
+- **SambaStudio (Option 2)**: Follow the instructions [here](../README.md#use-sambastudio-option-2) to set up your endpoint and environment variables.
+    Then, in the [config file](./config.yaml), set the llm `api` variable to `"sambastudio"`, and set the `CoE` and `select_expert` configs if you are using a CoE endpoint.
+
+### Set up the embedding model
+
+You have the following options to set up your embedding model:
+
+* **CPU embedding model (Option 1)**: In the [config file](./config.yaml), set the variable `type` in `embedding_model` to `"cpu"`.
+
+* **SambaStudio embedding model (Option 2)**: To increase inference speed, you can use a SambaStudio embedding model endpoint instead of using the default (CPU) Hugging Face embedding. Follow the instructions [here](../README.md#use-sambastudio-embedding-option-2) to set up your endpoint and environment variables. Then, in the [config file](./config.yaml), set the variable `type` in `embedding_model` to `"sambastudio"`, and set the configs `batch_size`, `coe` and `select_expert` according to your SambaStudio endpoint.
+
+## Windows requirements
+
+- If you are using Windows, make sure your system has Microsoft Visual C++ Redistributable installed. You can install it from [Microsoft Visual C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/) and make sure to check all boxes regarding C++ section. (Compatible versions: 2015, 2017, 2019 or 2022)
+
+# Deploy the starter kit GUI
+
+We recommend that you run the starter kit in a virtual environment or use a container. We also recommend using Python >= 3.10 and < 3.12.
+
+## Option 1: Use a virtual environment
+
+If you want to use virtualenv or conda environment:
+
+1. Install and update pip.
+
+    ```bash
+    cd ai_starter_kit/enterprise_knowledge_retriever
```

Comment:
This is EKR stuffs. Need to be updated with financial stuffs. Same thing for other parts of the README